.github/workflows/container-tests.yml

```diff
@@ -0,0 +1,88 @@
+name: Container-tests by GitHub Action at Testing Farm
+
+on:
+  issue_comment:
+    types:
+      - created
+jobs:
+  build:
+    # This job only runs for '[test]' pull request comments by owner, member
+    name: Container-tests by GitHub Action on Testing Farm service
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - tmt_plan: "fedora"
+            os_test: "fedora"
+            context: "Fedora"
+            compose: "CentOS-7"
+          - tmt_plan: "centos7"
+            os_test: "centos7"
+            context: "CentOS7"
+            compose: "CentOS-7"
+          - tmt_plan: "rhel7-docker"
+            os_test: "rhel7"
+            context: "RHEL7"
+            compose: "RHEL-7.9-Released"
+          - tmt_plan: "rhel8-docker"
+            os_test: "rhel8"
+            context: "RHEL8"
+            compose: "RHEL-8.3.1-Released"
+          - tmt_plan: "c9s"
+            os_test: "c9s"
+            context: "CentOS Stream 9"
+            compose: "CentOS-Stream-9"
+
+    if: |
+      github.event.issue.pull_request
+      && (contains(github.event.comment.body, '[test]') || contains(github.event.comment.body, '[test-all]'))
+      && contains(fromJson('["OWNER", "MEMBER"]'), github.event.comment.author_association)
+    steps:
+      - name: Get pull request number
+        id: pr_nr
+        run: |
+          PR_URL="${{ github.event.comment.issue_url }}"
+          echo "::set-output name=PR_NR::${PR_URL##*/}"
+
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          ref: "refs/pull/${{ steps.pr_nr.outputs.PR_NR }}/head"
+
+      - name: SHA value
+        id: sha_value
+        run: |
+          echo "::set-output name=SHA::$(git rev-parse HEAD)"
+
+      - name: Setup Testing Farm values
+        id: tf_values
+        run: |
+          branch_name="master"
+          api_key="${{ secrets.TF_INTERNAL_API_KEY }}"
+          tmt_repo="https://gitlab.cee.redhat.com/platform-eng-core-services/sclorg-tmt-plans"
+          if [ "${{ matrix.tmt_plan }}" == "fedora" ] || [ "${{ matrix.tmt_plan }}" == "centos7" ] || [ "${{ matrix.tmt_plan }}" == "c9s" ]; then
```

Comment:
In nginx's build-and-push-action were used the secrets names in the matrix:
https://github.com/sclorg/nginx-container/blob/34f118837f52307ed9abb7e7d5af248729ee11bf/.github/workflows/build-and-push.yml#L22
I would be happier if we use this solution also here, as we would get rid of one condition.
WDYT?

Reply:
I have missed that

Reply:
As I think about it again, IMHO whole "Setup Testing Farm values" section is not needed, if the values are assigned in the matrix. Or what is the reason, that this step is present?

Reply:
I wanted to have a shorter matrix and have one more "Setup Testing Farm values". But let's define everything in the matrix.